starter-code/styles/modules.css

```diff
@@ -10,9 +10,13 @@
 }
 
 .icon-menu {
-  /* TODO: Add styles for the icon-menu class (whether it be the
+  /* DONE: Add styles for the icon-menu class (whether it be the
       hamburger font or an image of your choosing). It should be
       bigger, and off to the right. */
+  float: right;
+  color: gray;
+  padding-top: 20px;
+  padding-right: 40px;
```

Comment:
Good work. Since we are dealing with icon fonts, we can use the font-size property just as if we were dealing with regular text 👍